app.js

```diff
@@ -302,7 +389,7 @@ async function watchRandomStreamers(browser, page) {
 
     info(`Account status: ${status}`);
     info(`Time: ${dayjs().format("HH:mm:ss")}`);
-    info(`Watching stream for ${watchminutes} minutes`);
+    info(`Watching stream for ${watchminutes} minutes => ${dayjs().add((watchminutes), 'minutes').format('HH:mm:ss')}`);
```

Comment:
I'd rather we used something like "ETA" instead of "=>"

Reply:
Ok, well your choice then :D